packages/react-instantsearch-dom/src/widgets/MenuSelect.js

```diff
@@ -46,4 +46,4 @@ const MenuSelectWidget = (props) => (
   </PanelCallbackHandler>
 );
 
-export default connectMenu(MenuSelectWidget);
+export default connectMenu(MenuSelectWidget, { $$widgetType: 'ais.menu' });
```

Comment:
I have `ais.menuSelect` in IS.js. Do we want to merge it with the menu widget?

```suggestion
export default connectMenu(MenuSelectWidget, { $$widgetType: 'ais.menuSelect' });
```